rabbit_monitor.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+import logging
+import sys
+from time import sleep
+
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from structlog import wrap_logger
+
+import settings
+
+__version__ = "0.0.1"
+
+logging.basicConfig(level=settings.LOGGING_LEVEL,
+                    format=settings.LOGGING_FORMAT)
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+class RabbitMonitor():
+
```

Comment:
Looks wrong to have all-caps Class variables.
Lots of other ways to do it, but how about defining a namedtuple with all the settings required, then populating `self.settings` in `__init__`.